tensorflow/python/feature_column/feature_column_v2_test.py

```diff
@@ -3419,8 +3419,8 @@ def _initializer(shape, dtype, partition_info):
     self.assertEqual(1 + 3 + 5, net.shape[1])
     with _initialized_session() as sess:
 
-      # Each row is formed by concatenating `embedded_body_style`,
-      # `one_hot_body_style`, and `price` in order.
+      # Each row is formed by concatenating `one_hot_body_style`,
```

Comment:
I think the original comment was fine. We sort the feature column names and this seems okay to me.